pwiz/data/msdata/Serializer_MGF.cpp

```diff
@@ -78,22 +78,22 @@ void Serializer_MGF::Impl::write(ostream& os, const MSData& msd,
     SpectrumList& sl = *msd.run.spectrumListPtr;
     SpectrumWorkerThreads spectrumWorkers(sl, useWorkerThreads, continueOnError);
     for (size_t i=0, end=sl.size(); i < end; ++i)
-    {
-        SpectrumPtr s;
-        try
-        {
-            // s = sl->spectrum(i, true);
-            s = spectrumWorkers.processBatch(i);
-        }
-        catch (std::exception& e)
-        {
-            if (continueOnError)
-            {
+    {
+        SpectrumPtr s;
```

Comment:
Revert whitespace only change.

Reply:
okay.

Reply:
I'm still seeing whitespace changes here. You know how to review the diff in GitHub? You can also review it locally if you set your diff engine to show whitespace changes.

Reply:
yes, sorry, but unfortunately it's not finished yet. Yes, It's easy to see whitespace changes, but undoing them without removing other important things is a bit difficult for me. But I´m on it. Thank you.

Reply:
Whitespaces should now be removed.